Show the actual month of the selected date

whoislyuboanyway opened the pull request.

whoislyuboanyway committed:
Show border instead of filled background

whoislyuboanyway committed:
# Show the actual month of the selected date

whoislyuboanyway committed:
Merge pull request #1 from whoislyuboanyway/show-border-instead-of-filled-background

Show the actual month of the selected date

whoislyuboanyway commented:
dev note: I implemented the missing functionality to show the correct month for the selected date. Please, keep in mind, that this is quick solution, and you can close the PR if you think it's not worth it :)

whoislyuboanyway committed:
Update  version

whoislyuboanyway committed:
Update README.md

shoheiyokoyama commented on Koyomi/DateModel.swift:
```diff
@@ -8,7 +8,16 @@
-public enum MonthType { case previous, current, next }
+public enum MonthType: Equatable { case previous, current, next, specific(diff: Int) }
+public func ==(lhs: MonthType, rhs: MonthType) -> Bool {
+    switch (lhs, rhs) {
```
You should define this function in `MonthType`.

shoheiyokoyama commented on Koyomi.podspec:
```diff
@@ -8,7 +8,7 @@
   s.name             = 'Koyomi'
-  s.version          = '1.2.7'
+  s.version          = '1.2.8'
   s.summary          = 'Simple customizable calendar component in Swift'
```
This change shouldn't be included in this pull request.

shoheiyokoyama commented:
@whoislyuboanyway 
Thank you for contributing 👍. 
Can you describe the contents of the bug in more detail in this pr or related issues?

whoislyuboanyway commented:
This is not a bug, I just added a specific enum case, where the month skip can be more than one month prior/after :) I needed it for the project I am working on, so I updated it.

p.s. so far I always moved the Equatable method outside of the equatable object :)
